packages/minifiers/minifiers.js

```diff
@@ -109,7 +109,7 @@ var rewriteRules = function (rules) {
     // a folder starting with "/packages/" is served on the same path as
     // it was on the filesystem and everything else is served on root "/".
     if (! basePath.match(/^\/?packages\//i))
-        basePath = "/";
+        basePath = (process.env.ROOT_URL_PATH_PREFIX || "") + "/";
```

Comment:
@glasser, I've done a few testings here and realized that there is no need to set the ROOT_URL_PATH_PREFIX env variable as it is already extracted from ROOT_URL in  `packages/meteor/url_server.js`.

The fix would be done in `packages/minifiers/minifiers.js` by using the `Meteor._relativeToSiteRootUrl` function:

    basePath = Meteor._relativeToSiteRootUrl(basePath);

But `packages/minifiers/minifiers.js` seems to be loaded before `packages/meteor/url_server.js` as the `ROOT_URL` is not set yet (and `__meteor_runtime_config__` is undefined on `minifiers.js`).

To overcome this situation I duplicated the `url_server.js` code and the Meteor._relativeToSiteRootUrl function into the `minifiers.js` and then it worked pretty well.

I used the todos example as a test (as it loads a few fonts) and I'm preparing a fork to exemplify this tweak, but the point is, why **Meteor._relativeToSiteRootUrl** is not ready when `minifiers.js` run?